chapter_assets/email-templates/email-templates.md

```diff
@@ -0,0 +1,210 @@
+#Email templates
+
+## New Sponsor Information
+Hi XXX,
+
+I'm writing you to give you more detailed information about codebar.io and provide you with details for hosting a workshop.
+
+**About codebar**
+
+Codebar.io is an organization that started in the UK. The aim is to help women and other underrepresented groups in tech learn to program.
+
+We do this by pairing students with volunteer developers who work at different companies throughout Barcelona. We started a chapter in Barcelona over two years ago, and host workshops every two weeks. https://www.meetup.com/Codebar-Barcelona/
+
+Some of our students want to become full-time developers, and others are interested in learning the basics in a welcoming environment. Our community is expanding and we are looking for more companies to host us.
+
+**How to host a workshop**
+
+If you are interested in hosting a workshop, this is what we would need. A space that can host a minimum of 35 people. Some tables to work at, an internet connection, power outlets, or extension cables to plug in laptops, and since people are coming to the workshops after work we encourage hosts to provide food and drinks. We ask that vegan options be available as well.    
```

Comment:
```suggestion
If you are interested in hosting a workshop, this is what we would need. A space that can host a minimum of 35 people. Some tables to work at, WIFI, power outlets, or extension cables to plug in laptops, and since people are coming to the workshops after work we encourage hosts to provide food and drinks. We ask that vegan options be available as well.    
```